Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,53 @@
-﻿namespace Logger;
+﻿using System;
+using System.Data.Common;
 
-public static class BaseLoggerMixins
+namespace Logger
 {
-    
+    public static class BaseLoggerMixins
+    {
+        public static void Error(this BaseLogger logger, string input, int parameter)
```

Comment:
`parameter` is supposed to be a parameter array so we can pass in a variable number of arguments.
```suggestion
        public static void Error(this BaseLogger logger, string input, params object[] args)
```